pages/hydra/1_new_start_here/other-getting-started.md

```diff
@@ -4,7 +4,23 @@ keywords: Dive Into Hydra Works
 permalink: /other-getting-started.html
 folder: hydra/getting_started/other-getting-started-docs.md
 tags: [getting_started]
+toc: false
 ---
 
+- [Hyrax ](https://github.com/projecthydra-labs/hyrax#getting-started)
 
-{% include links.html %}
+- [Solr](http://lucene.apache.org/solr/quickstart.html)
+
+- [Fedora 4 ](https://wiki.duraspace.org/display/FEDORA471/Quick+Start)
+
+- [Blacklight](https://github.com/projectblacklight/blacklight/wiki/Quickstart)
+
+- [Rails ](http://guides.rubyonrails.org/getting_started.html)
+
+- [AWS](https://aws.amazon.com/getting-started/)
```

Comment:
Well, it's a page for extra resources, not the main Up-and-Running-with-Hyrax one. People could be new to Hyrax but not to containers or Amazon hosting, or new to Ruby and Rails but not to Docker and Amazon, or familiar with Rails but new to Hyrax, Docker and Amazon.